lib/digitizer_block_impl.cc

```diff
@@ -700,9 +700,19 @@ namespace gr {
        }
      } catch (const std::exception& ex) {
        d_configure_exception_message = ex.what();
+       GR_LOG_ERROR(d_logger, d_configure_exception_message );
+
+       // It looks like the block is run by gnuradio, no matter if true or false is returned here
+       // TODO: "start" is called by gnuradio itself. Check if it is possible to stop the block from running when false is returned here.
```

Comment:
Gnuradio's own modules all return true, but the `tcp_sink` block just throws, so maybe gnuradio will actually catch the exception by itself and stop flowgraph execution and we can just rethrow the exception? Could also be an oversight in the gr_networking module because I haven't seen it anywhere else.
https://github.com/gnuradio/gnuradio/blob/7712360ddbe262ed6a48d622a695aaaaa4993795/gr-network/lib/tcp_sink_impl.cc#L76